web/src/components/pages/home/home.tsx

```diff
@@ -18,59 +19,52 @@ class Home extends React.Component<RouteComponentProps<any>, State> {
     const { showWallOfText } = this.state;
     return (
       <div id="home-container">
-        <h2 id="home-title">
-          The Common Voice project is Mozilla's initiative to help teach
-          machines how real people speak.
-        </h2>
+        <Localized id="home-title">
+          <h2 id="home-title" />
+        </Localized>
         <div
           id="wall-of-text"
           className={showWallOfText ? 'show-more-text' : ''}>
           <CardAction id="contribute-button" to="/record">
             <div>
               <RecordIcon />
             </div>
-            Speak up, contribute here!
+            <Localized id="home-cta">
+              <span />
+            </Localized>
           </CardAction>
 
-          <p>
-            Voice is natural, voice is human. That’s why we’re fascinated with
-            creating usable voice technology for our machines. But to create
-            voice systems, an extremely large amount of voice data is required.
-          </p>
+          <Localized id="wall-of-text-start">
+            <p />
+          </Localized>
 
-          <p className="more-text-mobile">
-            Most of the data used by large companies isn’t available to the
-            majority of people. We think that stifles innovation. So we’ve
-            launched Project Common Voice, a project to help make voice
-            recognition open to everyone.
-          </p>
+          <Localized id="wall-of-text-more-mobile">
+            <p className="more-text-mobile" />
+          </Localized>
 
-          <p className="more-text-desktop">
-            Now you can donate your voice to help us build an open-source voice
-            database that anyone can use to make innovative apps for devices and
-            the web. <br />
-            Read a sentence to help machines learn how real people speak. Check
-            the work of other contributors to improve the quality. It’s that
-            simple!
-          </p>
+          <Localized id="wall-of-text-more-desktop" $lineBreak={<br />}>
```

Comment:
In the new version of `fluent-react` this will just become `<Localized id="wall-of-text-more-desktop" br={<br />}>` and the translation can read:

```properties
wall-of-text-more-desktop =
    Now you can donate your voice to help us build an open-source voice database that anyone can use
    to make innovative apps for devices and the web.<br>
    Read a sentence to help machines learn how real people speak. Check the work of other
    contributors to improve the quality. It’s that simple!
```

The reason why this new line requires special handling is that even though Fluent preserves it when parsing the message, HTML normalizes it to a single space. Ideally, these would be two separate `<p>` elements. :)